### ISSUE TITLE
Documentation for %labels uses equals sign (wrong)

### ISSUE THREAD

**Issue by Flamefire** [apptainer/singularity#3862]
At https://sylabs.io/guides/3.2/user-guide/environment_and_metadata.html#environment-and-metadata it is shown:

```
Bootstrap: library
From: default/alpine

%labels
    OWNER = Joana
```

The equals sign is a copy&paste error. It should be omitted.

Furthermore the documentation should specify that the key-value pair is split at the first space. A good point to do that would be here: https://sylabs.io/guides/3.2/user-guide/definition_files.html#labels

**Comment by jchavezlavalle** [comment]
Fixed here: https://github.com/sylabs/singularity-userdocs/pull/207 (v3.2) and also here for latest documentation version: https://github.com/sylabs/singularity-userdocs/pull/206

**Comment by jchavezlavalle** [comment]
@jscook2345 already merged on both versions 3.2 and 3.3 docs, this fix was also approved by the OP on https://github.com/sylabs/singularity-userdocs/pull/207
